src/microsoft-extensions-ai/abstraction-implementations/AbstractionImplementationExamples/LoggingEmbeddingGenerator.cs

```diff
@@ -0,0 +1,23 @@
+using Microsoft.Extensions.AI;
+using Microsoft.Extensions.Logging;
+
+public class LoggingEmbeddingGenerator : DelegatingEmbeddingGenerator<string, Embedding<float>>
```

Comment:
Ditto

Reply:
Tracking in this issue https://github.com/dotnet/ai-samples/issues/79